doc/tutorials/content/sources/extract_indices/extract_indices.cpp

```diff
@@ -50,9 +50,9 @@ main (int argc, char** argv)
   // Create the filtering object
   pcl::ExtractIndices<pcl::PointXYZ> extract;
 
-  int i = 0, nr_points = (int) cloud_filtered->points.size ();
+  int i = 0, nr_points = (int) cloud_filtered->size ();
```

Comment:
nr_points as `auto`. Might require updating the lines in the corresponding tutorial.

Reply:
This falls under a more generic update for the tutorials, doesn't it?

Reply:
I don't understand the question. Can you be more explicit about what you are proposing?

Reply:
I meant that this isn't related to this PR. Current code has correct casts for the code to work